src/Lib/Utility/DataTypeConversion.php

```diff
@@ -22,7 +22,7 @@ public static function toType(string $cakeType): string
             // openapi type => ['cake','types']
             'integer' => ['int','integer','tinyinteger','smallinteger','biginteger','mediuminteger'],
             'number' => ['decimal','float'],
-            'string' => ['uuid','text','varchar','char','date','time','datetime','timestampfractional','timestamp'],
+            'string' => ['uuid','text','varchar','char','date','time','datetime','timestampfractional','timestamp','timestamptimezone'],
```

Comment:
The static analyzer has been angered. Break this into a multi-line array please.

Reply:
Apologies, it was originally a cheeky edit in the GitHub UI...

Reply:
NP, they are here to help us.